sqlglot/parser.py

```diff
@@ -1244,6 +1252,56 @@ def _parse_table_sample(self):
             seed=seed,
         )
 
+    def _parse_pivots(self, this):
```

Comment:
this could become

def _parse_pivots(self):
     return list(iter(self._parse_pivot, None))